src/main.ts

```diff
@@ -36,13 +36,13 @@ import { ShowOmniSharpConfigChangePrompt } from './observers/OptionChangeObserve
 import createOptionStream from './observables/CreateOptionStream';
 import { CSharpExtensionId } from './constants/CSharpExtensionId';
 import { OpenURLObserver } from './observers/OpenURLObserver';
-import { activateRazorExtension } from './razor/razor';
-import { RazorLoggerObserver } from './observers/RazorLoggerObserver';
-import { AbsolutePathPackage } from './packageManager/AbsolutePathPackage';
-import { downloadAndInstallPackages } from './packageManager/downloadAndInstallPackages';
-import IInstallDependencies from './packageManager/IInstallDependencies';
-import { installRuntimeDependencies } from './InstallRuntimeDependencies';
-import { isValidDownload } from './packageManager/isValidDownload';
+//import { activateRazorExtension } from './razor/razor';
```

Comment:
bunch of code to comment out anything that doesnt work on the web extension version. in hindsight I should have started adding only what I needed, but....